src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/DesignTimeInputs.cs

```diff
@@ -7,13 +7,13 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.TempPE
 {
     internal class DesignTimeInputs
     {
-        public readonly ImmutableArray<string> Inputs;
-        public readonly ImmutableArray<string> SharedInputs;
+        public readonly ImmutableHashSet<string> Inputs;
+        public readonly ImmutableHashSet<string> SharedInputs;
```

Comment:
Properties here?

Reply:
Thanks, my bad. This file wasn't in your first review.